ApiBundle/Controller/ContentTypeController.php

```diff
@@ -63,11 +63,17 @@ public function listAction(Request $request)
         );
 
         $repository = $this->get('open_orchestra_model.repository.content_type');
+        $transformer = $this->get('open_orchestra_api.transformer_manager')->get('content_type_collection');
+
+        if ($request->get('entityId')) {
```

Comment:
ConteTypeController has never used the Trait because of the different query used to set collection.